code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -231,6 +231,19 @@
 		)
 	twohanded = FALSE
 
+/obj/item/weapon/gun/energy/psychic/mindflayer
+	name = "Prototype: mind flayer"
+	desc = "A cruel weapon designed to break the minds of those it targets, causing sanity loss and mental breakdowns."
+	icon = 'icons/obj/guns/energy/xray.dmi'
+	icon_state = "xray"
+	projectile_type = /obj/item/projectile/beam/psychic
+	fire_sound = 'sound/weapons/Laser.ogg'
+	slot_flags = SLOT_BELT|SLOT_BACK
+	fire_delay = 10
+	price_tag = 3000
+	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_SILVER = 5, MATERIAL_PLASMA = 3)
+	twohanded = FALSE
```

Comment:
Is it a two-handed or one-handed? I don't think it should go on `SLOT_BELT` if it's pistol(normal)-sized. If it's not it may as well be twohanded.

Reply:
It's one handed, same as the psionic laser cannon. I don't really get what you mean by the rest.

Reply:
I've actually meant it should go in holster if it's pistol-sized, not vice versa. w/e